src/components/MainSection/MainSec.css

```diff
@@ -1,4 +1,27 @@
 .main-section {
     color: #000;
     background-color: #fff;
-}
\ No newline at end of file
+}
+.top-btn{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    position: fixed;
+    right: 4rem;
+    bottom: 4rem;
+    border-radius: 15px;
+    color: #ffffff;
+    background-color: #121916;
+    height: 3.5rem;
+    width: 3.5rem;
+    font-size: 1.3rem;
+    z-index: 999;
+}
+
+@media screen and (max-width:700px){
+    .top-btn{
+        opacity: 0;
```

Comment:
making `opacity: 0` will only make it invisible but the element is present there everytime. 
Try display property instead.

```
display: none;
```